op-node/node/node.go

```diff
@@ -203,8 +203,10 @@ func (n *OpNode) initL1(ctx context.Context, cfg *Config) error {
 	// which only change once per epoch at most and may be delayed.
 	n.l1SafeSub = eth.PollBlockChanges(n.log, n.l1Source, n.OnNewL1Safe, eth.Safe,
 		cfg.L1EpochPollInterval, time.Second*10)
-	n.l1FinalizedSub = eth.PollBlockChanges(n.log, n.l1Source, n.OnNewL1Finalized, eth.Finalized,
-		cfg.L1EpochPollInterval, time.Second*10)
+	if !cfg.Plasma.Enabled {
```

Comment:
Needs a comment explaining that it doesn't exist / where `n.OnNewL1Finalized` is called.